.github/workflows/lra-test.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: LRA test
+
+on:
+  push:
+    branches: [ master ]
+    paths:
+      - "rts/lra/**"
+  pull_request:
+    branches: [ master ]
+    paths:
+      - "rts/lra/**"
+
+jobs:
+  lra-test:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Narayana - build
+      run: ./build.sh clean install -Pcommunity -DskipTests -Didlj-enabled=true
+    - name: LRA - test (tck/basic)
+      run: |
+        cd rts/lra/lra-test
+        mvn clean install -Parq
```

Comment:
If you aren't going to use narayana.sh, please can you use the maven in the Narayana repo though still?

I am not sure though whether moving away from narayana.sh is good to be able to keep the GH action and the existing CI running in parallel with the same testing /cc @ochaloup

Reply:
I can reuse narayana.sh but that would be only running a single line at https://github.com/jbosstm/narayana/blob/master/scripts/hudson/narayana.sh#L483 if I am not missing anything. This would just require a bunch of env properties to be set in this yaml directly. Do you want me to change to that? Also it would need to branch the command because we don't want timeout adjustment that is included in the Hudson CI.

Reply:
In this branch of the discussion I am discussing if we _weren't_ to use narayana.sh. So in this case it would be something like changing the line `mvn clean install -Parg` to (I guess, based on the path): `../../../build.sh clean install -Parq` (or remove the `cd` and go with `./build.sh -f rts/lra/lra-test/pom.xml clean install -Parq`

It's not very clear to me where the `mvn` is coming from in the command as it stands, maybe it's part of `ubuntu-latest`? I think for reproducibility though we should be using the Maven in narayana: https://github.com/jbosstm/narayana/tree/master/tools/maven by calling https://github.com/jbosstm/narayana/blob/master/build.sh

Does that help clarify?

Reply:
this won't work with TT I believe as it's documented in https://github.com/jbosstm/narayana/blob/master/scripts/hudson/narayana.sh#L480.